Core/DISET/private/Transports/SSL/M2Utils.py

```diff
@@ -47,6 +51,18 @@ def __loadM2SSLCTXProxy(ctx, proxyPath=None):
   ctx.load_cert_chain(proxyPath, proxyPath, callback=lambda: "")
 
 
+def ssl_verify_callback_print_error(ok, store):
+  """ This callback method does nothing but printing the error.
+      It prints a few more useful info than the exception
+    :param ok: current validation status
+    :param store: pointer to the X509_CONTEXT_STORE
```

Comment:
I think this will fix the docs:
```suggestion
  """This callback method does nothing but printing the error.
     It prints a some more useful information than just the exception.

     :param ok: current validation status
     :param store: pointer to the X509_CONTEXT_STORE
```

Reply:
Argh, probably ! Thanks